src/pages/manager/OpenAccountPage.js

```
@@ -1,13 +1,38 @@
 import { expect } from '@playwright/test';
 
-export class AddCustomerPage {
+export class OpenAccountPage {
   constructor(page) {
     this.page = page;
+    this.currency = page.locator('select#currency')
+    this.customer = page.locator('select#userSelect')
+    this.processButton = page.locator('button').filter({hasText:'Process'})
   }
 
-  async open() {
+  async openManagerAccount() {
     await this.page.goto(
       '/angularJs-protractor/BankingProject/#/manager/openAccount',
     );
   }
+
+  async waitForOpenManagerAccount () {
+    await this.page.waitForURL('/angularJs-protractor/BankingProject/#/manager/openAccount')
+  }
+
+  async selectCurrency (money) {
+    await this.currency.selectOption({ label: money});
+  }
+
+ async assertCurrency(expectedLabel) {
+  const selectedText = await this.currency.locator('option:checked').textContent()
+  expect(selectedText.trim()).toBe(expectedLabel)
+}
+
+  async selectCustomer (person) {
+    await this.customer.selectOption({ label: person })
```

selectCustomer uses `selectOption({ label: person })` which is the correct approach when tests provide the visible label. Add a wait for the customer select to be visible/attached before selecting to avoid timing issues, e.g.: `await this.customer.waitFor({ state: 'visible' })` prior to `selectOption`. Also consider asserting the selection succeeded (e.g. check `option:checked` text) to fail fast with clearer message if the provided name is not in the list.